### DISCUSSION TITLE
Does AGIC work with IPv6 application gateway ?

### DISCUSSION THREAD

**Discussion by AzureWithBen** [Azure/application-gateway-kubernetes-ingress#1583] [Q&A]
Hello guys,

I'm wondering if the agic works with the preview dual-stack (IPv4 + IPv6)  feature of the Application Gateway ?
How the AGIC will know to create listener for both IPv4 and IPv6 Public IP ? Do I need to use 2 AGIC, one for each public IP ?

Thank you for your answer :).

Cheers

**Comment by rg54** [comment]
Hello @BenoitFi 
I've asked the same question, and found your question when searching for info.
[According to MS documentation](https://learn.microsoft.com/en-us/azure/application-gateway/ipv6-application-gateway-portal), it is not supported to date : 
![image](https://github.com/user-attachments/assets/6fd1a908-2f32-41b8-a075-7112bf412020)

And I haven't found an issue about this, I think we should create one.
